src/panels/media-browser/hui-dialog-select-media-player.ts

```diff
@@ -84,8 +88,8 @@ export class HuiDialogSelectMediaPlayer extends LitElement {
         ha-dialog {
           --dialog-content-padding: 0 24px 20px;
         }
-        paper-item {
-          cursor: pointer;
+        mwc-list-item[noninteractive] {
+          color: var(--disabled-text-color);
         }
```

Comment:
```suggestion
```

Reply:
We should still show the disabled color?

So I want to either class map or ?disabled and use the attribute

Reply:
a disabled mwc-list should have disabled color, if not we should set the right vars.